README.md

```diff
@@ -58,7 +57,34 @@ deployment options.
 }
 ```
 
-### Adding new integrations
+#### Running storage operations locally
+If you need to run storage (S3, GCS or Azure Blob) operations locally you can run the agent setting these 
+environment variables: `MCD_STORAGE` and `MCD_STORAGE_BUCKET_NAME`, for example:
+```shell
+PYTHONPATH=. MCD_DEBUG=true MCD_STORAGE_BUCKET_NAME=agent-bucket MCD_STORAGE=GCS python apollo/interfaces/generic/main.py
```

Comment:
Not a bad idea to include a `.envrc.example` file or similar on the repo with all the env vars that you usually need to set, though they are not that many today.

BTW, instead of `PYTHONPATH=.` it also works to run `python -m apollo.interfaces.generic.main`